Modelica/Constants.mo

```diff
@@ -54,10 +59,12 @@ package Constants
 <p>
 This package provides often needed constants from mathematics, machine
 dependent constants and constants from nature. The latter constants
-(name, value, description) are from the following source:
+(name, value, description) are from the following source (will add updated codata reference when available):
```

Comment:
Is it available?

Reply:
It was easy to find that it should exist, but a bit hard to find - since none of the main-web-sites have a real reference; they are assumedly waiting for the update.
But finally I found the link in the http://codata.org/blog/category/fundamental-constants/ blog: 
https://iopscience.iop.org/article/10.1088/1681-7575/aa950a/pdf
Will add.